app/mutations/FundingParameter/createFundingParameterFormChange.ts

```diff
@@ -19,7 +19,6 @@ export const mutation = graphql`
           changeStatus
           projectRevisionByProjectRevisionId {
             ...TaskList_projectRevision
-            totalProjectValue
```

Comment:
Don't think we need calculated fields in here because the fields filled in on create aren't enough to trigger the calculation